src/views/List.jsx

```diff
@@ -1,13 +1,43 @@
 import { ListItem } from '../components/ListItem';
+import { useState } from 'react';
```

Comment:
Extremely pedantic but it's common practice to put library imports at the very top above files/components that are in the project.